### DISCUSSION TITLE
Add option to hibernate tabs on command

### DISCUSSION THREAD

**Discussion by Momen-MKadry** [Floorp-Projects/Floorp#910] [Feature Request]
I like the sleeping tabs feature because I open lots of tabs and having them hibernate after some giving time is useful, but there is this option that Vivaldi has where you assign a keyboard shortcut to hibernate background tabs whenever you want, I sometimes need this because I might open lots of tabs but still need them present but not active.

**Comment by Telmesomething** [comment]
There's a keyboard shortcut menu available with various options. To access it open Settings and then copy/paste: `about:preferences#cSK`

**Reply by Momen-MKadry** [reply]
I looked there and sadly this feature is not present there
